package.json

```diff
@@ -76,6 +78,8 @@
   "type": "module",
   "dependencies": {
     "@uswds/uswds": "^3.4.1",
-    "fast-blurhash": "^1.1.2"
+    "@contentful/rich-text-types": "^16.0.3",
+    "fast-blurhash": "^1.1.2",
+    "lodash": "^4.17.21"
```

Comment:
I know lodash isn't a huge dependency, and I'm not entirely clear on if we're doing any tree-shaking automatically, but if we're not, we may want to find a way to just get the `lodash/escape` function that we're using here (and maybe do something with `lodash/chunk` while we're at it). 

I haven't looked into this all that much, but I think I've found two ways of going about this:
1. Install  [lodash.escape](https://www.npmjs.com/package/lodash.escape) and [lodash.chunk](https://www.npmjs.com/package/lodash.chunk). My guess is that there would be some overlap between these two but that might be the price of convenience.
2. Create a [custom build](https://lodash.com/custom-builds) using a CLI where we run `lodash include=chunk,escape` to generate a much smaller package. This would probably create the smallest dependency size, but might be a pain to maintain.

All that said, if you're seeing that we're only including relevant files for `lodash/escape` in the build, you can completely ignore everything I've said above 😅 . Might be nice to replace our `chunk` implementation with `lodash/chunk` while you're here though.

Reply:
`lodash` exports a separate entrypoint for every function. If we imported `escape` like `import { escape } from "lodash"` we would have to verify that tree-shaking was working properly (it usually doesn't), but by importing it like `import escape from "lodash/escape"` we are ensuring that we are importing the minimal bundle to support `escape` (in fact, this is what plugins that claim to improve your lodash tree-shaking do, they rewrite imports to the `import defaultExport from lodash/*` form). See [this page in the Lodash docs](https://lodash.com/per-method-packages) for more details.

Reply:
I've replaced our custom "chunk" function with `lodash/chunk` as you suggested.